tokio/src/runtime/metrics/metrics.rs

```diff
@@ -0,0 +1,68 @@
+//! This file contains the types necessary to collect various types of metrics.
+use crate::loom::sync::atomic::{AtomicU64, Ordering};
+
+pub(crate) struct RuntimeMetrics {
```

Comment:
Will this struct have all the public accessors? Or do users interact with `WorkerMetrics`? Or both?

Might be worthwhile adding one example for such an accessor to see how its used.